lib/factgrp.gi

```diff
@@ -1727,6 +1733,45 @@ local s,r,nat,k;
   od;
 end);
 
+#############################################################################
+##
+#F  TryQuotientsFromFactorSubgroups(<hom>,<ker>,<bound>) 
+##
+InstallGlobalFunction(TryQuotientsFromFactorSubgroups,function(hom,ker,bound)
+local s,p,k,it,u,v,d,ma,mak,lev,sub,low;
+  s:=Source(hom);
+  p:=Image(hom);
```

Comment:
Observation: `TryQuotientsFromFactorSubgroups ` is never executed by the Travis test suite, which include testinstall and teststandard. Is it only used for gigantic example or something like that?